action.yml

```diff
@@ -25,7 +25,7 @@ runs:
       cd "${GITHUB_WORKSPACE}" || exit 1
       TEMP_PATH="$(mktemp -d)"
       PATH="${TEMP_PATH}:$PATH"
-      curl -sfL https://raw.githubusercontent.com/Songmu/tagpr/main/install.sh | sh -s -- -b "$TEMP_PATH" "${{ inputs.version }}" 2>&1
+      curl -sfL https://raw.githubusercontent.com/Songmu/tagpr/v1.5.2/install.sh | sh -s -- -b "$TEMP_PATH" "${{ inputs.version }}" 2>&1
```

Comment:
I am aware that this version value is automatically updated by https://github.com/Songmu/tagpr/blob/main/.tagpr#L20.

Reply:
ops.. https://github.com/Songmu/tagpr/blob/6f76534e4066713c6b49eac9aedd44887f9eb15d/action.yml 

It might be better to allow the version to be set via an environment variable.